### ISSUE TITLE
Importing new CA with passphrase causes server to hang while passphrase is asked in console

### ISSUE THREAD

**Issue by ferrine** [openwisp/django-x509#36]
I was playing with django-x509 before deciding to build a project on top of it. I've tried to import my existing CA in the admin panel (running on docker image) but got no response. After a closer look I've found PEM passphrase request for CA. That was really surprising because I had to type it in CLI (where logs were printed). Afterwards, problem was solved. It's probably better to have password field in the form and automatically pass it to verify CA

In short, I'm planning to import all my VPN clients into the app

**Comment by nemesisdesign** [comment]
So you're trying to import an existing CA which has a passphrase set.

This is currently not supported, I'm updating the title of the issue.

I'd prefer a `passphrase` field rather than `password` as I think it's more closely related to the x509 terminology.

**Comment by R9295** [comment]
I'll take this up.

**Comment by nemesisdesign** [comment]
@R9295 great! I suggest to try to replicate the issue first.

**Comment by nemesisdesign** [comment]
@ferrine the new feature is in the current master (`0.5.0 alpha`)
